examples/morpheusvm/tests/integration/integration_test.go

```diff
@@ -226,6 +265,7 @@ var _ = ginkgo.BeforeSuite(func() {
 			app,
 		))
 
+		server.Initialize(v)
```

Comment:
Gross pattern, but we should be able to delete all of this once we get grpc-based apis in.